test/onnx/test_fx_to_onnx_with_custom_op_overload.py

```diff
@@ -0,0 +1,56 @@
+# Owner(s): ["module: onnx"]
+import onnxscript
+import pytorch_test_common
+import torch
+import torch.onnx._internal.fx.custom_operator
+from torch.onnx._internal import fx
+from torch.testing._internal import common_utils
+
+
+class TestFxToOnnx(pytorch_test_common.ExportTestCase):
+    def setUp(self):
+        super().setUp()
+        self.opset_version = torch.onnx._constants.ONNX_DEFAULT_OPSET
+
+    def test_simple_function(self):
+        # Steps to put custom op into onnx model
+        #  1. Define a custom function and register it as torch._ops.OpOverload so that
+        #     this function can pass through PyTorch 2.0's FX-generation pipeline
+        #     (e.g., dynamo.export). Otherwise, the custom function will be traced
+        #     into (or decomposed) and exporter won't see any custom functions.
+        #  2. Write a custom exporter using ONNX Script for this custom function.
+        #  3. Register the custom exporter to the custom function (type: torch._ops.OpOverload).
+
+        # Step 1
+        @fx.custom_operator._register_onnx_custom_op_overload(
+            "torch_custom_op(Tensor x) -> Tensor"
+        )
+        def custom_op(x):
+            return x * 2 + 1
+
+        CUSTOM_OPSET = onnxscript.values.Opset(domain="com.custom", version=1)
+
+        # Step 2
+        @onnxscript.script(opset=CUSTOM_OPSET)
+        def custom_op_exporter(x):
```

Comment:
The name of the function becomes the local function name inside the onnx model. I suggest adding some emphasis to it in the doc. I guess it shouldn't be called `xx_exporter` ...

Reply:
`custom_op_exporter` is supposed to be the name of the op/function used in the ONNX model. And the body below is supposed to define the function-body of this op/function.

I am a little unclear about this example. May be I would understand it better if the body below did not call a custom op (but was defined in terms of standard onnx ops). As it is, this both defines a custom function-op and uses a custom op. I just wonder if that was the intention?

Reply:
@BowenBao, sure. I will improve the doc.

@gramalingam, it's indented. It is used to map PyTorch custom `op` to ONNX custom `op`. Therefore, the function body here needs to generate ONNX custom `op`.

Reply:
@wschin If it is always going to have this fixed form, why even force the user to do this? It seems to me that onnxscript functionality is not going to be used? I wonder if it is possible to do what you want, without asking the user to do this step 2.

Reply:
It's not always this form. It depends on how the author wants to map their custom op into their own opset. We can have a default form and allow users to override it.